core/files/file-types/svg.php

```diff
@@ -450,94 +450,182 @@ private function has_js_value( $value ) {
 	 */
 	private function get_allowed_attributes() {
 		$allowed_attributes = [
-			'class',
-			'clip-path',
-			'clip-rule',
-			'fill',
-			'fill-opacity',
-			'fill-rule',
-			'filter',
-			'id',
-			'mask',
-			'opacity',
-			'stroke',
-			'stroke-dasharray',
-			'stroke-dashoffset',
-			'stroke-linecap',
-			'stroke-linejoin',
-			'stroke-miterlimit',
-			'stroke-opacity',
-			'stroke-width',
-			'style',
-			'systemlanguage',
-			'transform',
-			'href',
-			'xlink:href',
-			'xlink:title',
-			'cx',
-			'cy',
-			'r',
-			'requiredfeatures',
-			'clippathunits',
-			'type',
-			'rx',
-			'ry',
-			'color-interpolation-filters',
-			'stddeviation',
-			'filterres',
-			'filterunits',
-			'height',
-			'primitiveunits',
-			'width',
-			'x',
-			'y',
-			'font-size',
-			'display',
-			'font-family',
-			'font-style',
-			'font-weight',
-			'text-anchor',
-			'marker-end',
-			'marker-mid',
-			'marker-start',
-			'x1',
-			'x2',
-			'y1',
-			'y2',
-			'gradienttransform',
-			'gradientunits',
-			'spreadmethod',
-			'markerheight',
-			'markerunits',
-			'markerwidth',
-			'orient',
-			'preserveaspectratio',
-			'refx',
-			'refy',
-			'viewbox',
-			'maskcontentunits',
-			'maskunits',
-			'd',
-			'patterncontentunits',
-			'patterntransform',
-			'patternunits',
-			'points',
-			'fx',
-			'fy',
-			'offset',
-			'stop-color',
-			'stop-opacity',
-			'xmlns',
-			'xmlns:se',
-			'xmlns:xlink',
-			'xml:space',
-			'method',
-			'spacing',
-			'startoffset',
-			'dx',
-			'dy',
-			'rotate',
-			'textlength',
+			'accent-height',
```

Comment:
Hi @filipkis 
Thanks for sending the list!
- Can you please add the credit for the list in the code?
- Can you fix the indentation of the list?

Reply:
@filipkis Any update?

Reply:
Added the credits and fixed the indentation.